crates/bitwarden-vault/src/cipher/cipher_client.rs

```diff
@@ -67,6 +70,26 @@ impl CiphersClient {
         Ok(credentials)
     }
 
+    /// Temporary method used to re-encrypt FIDO2 credentials for a cipher view.
+    /// Necessary while the FIDO2 credentials remain in the LoginView in an encrypted form.
+    /// Used by the TS clients to encrypt FIDO2 credentials separately before encrypting the
+    /// remaining cipher view. TODO: Remove once FIDO2 credentials have been removed from the
+    /// LoginView.
```

Comment:
Please add a jira ticket for the this work, would it just be to revert this commit?

Reply:
Added JIRA ticket number [d8e596b](https://github.com/bitwarden/sdk-internal/pull/331/commits/d8e596b01893d060c53648c07c03074a17c160ed)

And yes, we could just revert this PR.